Examples/Swift/UserLocationAnnotationExample.swift

```diff
@@ -81,14 +82,13 @@ class CustomUserLocationAnnotationView: MGLUserLocationAnnotationView {
             arrowSize = size / 2.5
             arrow = CAShapeLayer()
             arrow.path = arrowPath()
-            arrow.bounds = CGRect(x: 0, y: (size/2 + 2), width: arrowSize, height: arrowSize)
+            arrow.bounds = CGRect(x: 0, y: 0, width: arrowSize, height: arrowSize)
```

Comment:
Updated in [ccf36a5](https://github.com/mapbox/ios-sdk-examples/pull/50/commits/ccf36a5814ff3ce72c677418d255e9548decd98d)